lib/task-data/schemas/linux-command.json

```diff
@@ -43,7 +43,7 @@
                 "downloadUrl": {
                     "description": "Specify the download URL (relative to RackHD southbound API server) if the command needs to download some data or script from RackHD before execution",
                     "type": "string",
-                    "pattern": "^/[a-zA-Z0-9+-.#?=\\\/]+"
+                    "minLength": 1
```

Comment:
I would suggest to keep the ```pattern``` and check ```downloadurl``` against a valid http URL pattern. There are talks/PRs around changing relative URI to fullpath URL and this check will help people catch errors if a legacy relative URI is used.

Ref: https://github.com/RackHD/RackHD/issues/489#issuecomment-262570418